src/common/utils/search.utils.ts

```diff
@@ -46,7 +41,10 @@ export type orderFilterType = {
   [key: string]: orderFilterType | string;
 };
 
-export function orderFilter(order: string[]) {
+export function orderFilter(order: string[] | undefined): orderFilterType[] {
+  if (order == undefined) {
```

Comment:
```suggestion
  if (!order) {
```
얘도 falsy?

Reply:
동일한 상황입니다.
https://github.com/sparcs-kaist/otlplus-server/pull/46#discussion_r1335200900

Reply:
위 코멘트 이해했고, 그러면 
```suggestion
  if (order === undefined) {
```
strict equal로만 수정하시죵

Reply:
앗 전 바보입니다